src/main/utils/utils.ts

```diff
@@ -0,0 +1,161 @@
+import * as pathUtils from 'path';
+import * as vscode from 'vscode';
+import * as os from 'os';
+import * as fs from 'fs';
+import AdmZip, { IZipEntry } from 'adm-zip';
+import { ScanUtils } from './scanUtils';
+
+export class Utils {
+    private static readonly MAX_FILES_EXTRACTED_ZIP: number = 1000;
+    private static readonly MAX_SIZE_EXTRACTED_ZIP: number = 1000000000; // 1 GB
```

Comment:
Add the unit (bytes, right?)
```suggestion
    // 1 GB
    private static readonly MAX_SIZE_EXTRACTED_ZIP_BYTES: number = 1000000000; 
```